decidim.gemspec

```diff
@@ -42,6 +42,7 @@ Gem::Specification.new do |s|
   s.add_dependency "decidim-consultations", Decidim.version
   s.add_dependency "decidim-core", Decidim.version
   s.add_dependency "decidim-debates", Decidim.version
+  s.add_dependency "decidim-initiatives", Decidim.version
```

Comment:
Sorry to pop in so late - I believe we should leave `initiatives` outside the main gem unless we have a way to toggle initiatives for an organization. Otherwise, we might be introducing a feature some people might not desire.